apps/daimo-mobile/app.json

```diff
@@ -79,7 +79,10 @@
           "faceIDPermission": "Protect Daimo using Face ID."
         }
       ],
-      ["expo-notifications"]
+      [
+        "expo-notifications"
+      ],
+      "react-native-nfc-manager"
```

Comment:
You should add prompt strings in [info.plist](https://docs.expo.dev/versions/latest/config/app/#infoplist) as described in https://github.com/revtel/react-native-nfc-manager#setup

Reply:
Added it in this commit: https://github.com/daimo-eth/daimo/pull/305/commits/47d81118af0069e8a37f72aef7e7621e1b434220

Android doesn't need one, as its not a runtime permission